5_databases/week2/hwkpart2data.sql

```diff
@@ -0,0 +1,42 @@
+-- Active: 1706445451493@@127.0.0.1@3306@School_database
+CREATE TABLE school_class (
+    id INT PRIMARY KEY AUTO_INCREMENT,
+    name VARCHAR(255) NOT NULL,
+    start_date DATE,
+    end_date DATE,
+    status ENUM('not-started', 'on-going', 'finished') DEFAULT 'not-started'
```

Comment:
Great work on these `CREATE`-statements! 🎉 I think the homework intended for you to add the `status` column through an `ALTER`-statement. How would that look?